frontend/packages/operator-lifecycle-manager/src/components/operand.tsx

```diff
@@ -279,7 +282,9 @@ export const ProvidedAPIsPage = connect(inFlightStateToProps)((props: ProvidedAP
   const createProps =
     owned.length > 1
       ? {
-          items: owned.reduce((acc, crd) => ({ ...acc, [crd.name]: crd.displayName }), {}),
+          items: owned
+            .filter((item) => !isInternalObject(internalObjects, item.name))
+            .reduce((acc, crd) => ({ ...acc, [crd.name]: crd.displayName }), {}),
```

Comment:
Reduce can be used to filter. No need to filter and then reduce. We can expand the logic in the reduce callback to exclude items that should be filtered.

Reply:
ack